manifest.php

```diff
@@ -85,5 +86,8 @@
     ],
     'extra' => [
         'structures' => $extpath . 'controller' . DIRECTORY_SEPARATOR . 'structures.xml',
-    ]
+    ],
+    'containerServiceProviders' => [
```

Comment:
Please ensure `core` and `generis` versions in `composer.json` matches this new feature

Reply:
Only `generis` should be bumped as I can see